bots/codeTemplate.json

```diff
@@ -0,0 +1,4 @@
+{
+  "execTemplate": "(async (bot) => {\n\n    /* CODE HERE */\n    log(bot, 'Code finished.');\n\n});",
+  "checkTemplate": "import * as skills from '../../../src/agent/library/skills.js';\nimport * as world from '../../../src/agent/library/world.js';\nimport Vec3 from 'vec3';\n\nconst log = skills.log;\n\nexport async function main(bot) {\n    /* CODE HERE */\n    log(bot, 'Code finished.');\n}"
```

Comment:
Can you return this to its original form, in two files `execTemplate.js` and `lintTemplate.js`. That would be much more readable. Sorry to go back on this, I now understand you can't combine them